records_mover/url/base.py

```diff
@@ -133,8 +155,9 @@ def download_fileobj(self, output_fileobj: IO[bytes]) -> None:
 
     def store_string(self, contents: str) -> None:
         "Save the specified string to the file."
-        with self.open(mode='wt') as f:
-            f.write(contents)
+
+        with self.open(mode='wb') as f:
+            f.write(contents.encode('utf-8'))
```

Comment:
The smartopen stuff only works with fileobjs opened in binary mode, so let's just do that.